From 01c9c5302602a9e2b1c853e5b6385f13acc69f5f Mon Sep 17 00:00:00 2001
From: tteras <tteras>
Date: Thu, 23 May 2013 05:44:28 +0000
Subject: [PATCH 6/9] From Rainer Weikusat <rweikusat@mobileactivedefense.com>:
 Fix SADB_X_EALG_CASTCBC definition to use system definition (which differs at
 least on Linux).

---
 src/include-glibc/net/pfkeyv2.h | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/include-glibc/net/pfkeyv2.h b/src/include-glibc/net/pfkeyv2.h
index e8eb4f8..47812d8 100644
--- a/src/include-glibc/net/pfkeyv2.h
+++ b/src/include-glibc/net/pfkeyv2.h
@@ -1,4 +1,4 @@
-/*	$NetBSD: pfkeyv2.h,v 1.4 2006/09/09 16:22:08 manu Exp $	*/
+/*	$NetBSD: pfkeyv2.h,v 1.4.40.1 2013/05/23 05:44:28 tteras Exp $	*/
 
 #ifndef __NET_PFKEYV2_H_
 #define __NET_PFKEYV2_H_ 1
@@ -18,7 +18,11 @@
 #define SADB_X_AALG_SHA              250
 
 /* private allocations - based on RFC2407/IANA assignment */
-#define SADB_X_EALG_CAST128CBC	5	/* SADB_X_EALG_CASTCBC? == 6 */
+#ifdef SADB_X_EALG_CASTCBC
+#define SADB_X_EALG_CAST128CBC		SADB_X_EALG_CASTCBC
+#else
+#define SADB_X_EALG_CAST128CBC		5
+#endif
 #define SADB_X_EALG_RIJNDAELCBC		SADB_X_EALG_AESCBC
 #define SADB_X_EALG_AES			SADB_X_EALG_AESCBC
 
-- 
1.8.4.3

